system_files/silverblue/usr/etc/profile.d/bluefin-fastfetch.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 
-alias fastfetch="/usr/bin/fastfetch --logo /usr/share/ublue-os/bluefin-logo -c /usr/share/ublue-os/ublue-os.jsonc"
-alias neofetch="/usr/bin/fastfetch --logo /usr/share/ublue-os/bluefin-logo -c /usr/share/ublue-os/ublue-os.jsonc"
\ No newline at end of file
+BLUEFIN_LOGO_PATH=/usr/share/ublue-os/bluefin-logos/symbols
+BLUEFIN_LOGO_RANDOM=$(shuf -e "$(ls -1 ${BLUEFIN_LOGO_PATH})" | head -1)
+BLUEFIN_FETCH_LOGO="${BLUEFIN_LOGO_PATH}/${BLUEFIN_LOGO_RANDOM}"
+
+#shellcheck disable=SC2139
+alias fastfetch="/usr/bin/fastfetch --logo ${BLUEFIN_FETCH_LOGO}  -c /usr/share/ublue-os/ublue-os.jsonc"
```

Comment:
Maybe we can make this run the shuffle each time so that you can run it more than once in a terminal session to see the other dinos. Makes it more fun. :) I'd put the randomization code in the alias.

Reply:
Not necessarily in this PR :-)